### PR TITLE
HtmlEditor - Use Table Formatting API instead of direct DOM element customization

### PR DESCRIPTION
**Pull request by alexander-kotov-dx**

### CODE REVIEW

File: js/ui/html_editor/modules/tableResizing.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ import Draggable from '../../draggable';
 import { each } from '../../../core/utils/iterator';
 import { getWindow, hasWindow } from '../../../core/utils/window';
 import { extend } from '../../../core/utils/extend';
-import { setLineElementsStyleValue, getLineElements, getAutoSizedElements, getColumnElements, unfixTableWidth } from '../utils/table_helper';
+import { setLineElementsFormat, getLineElements, getAutoSizedElements, getColumnElements, unfixTableWidth } from '../utils/table_helper';
```
Comment (DokaRus) [inline js/ui/html_editor/modules/tableResizing.js]:
```suggestion
import {
    setLineElementsFormat,
    getLineElements,
    getAutoSizedElements,
    getColumnElements,
    unfixTableWidth
} from '../utils/table_helper';
```

File: js/ui/html_editor/modules/tableResizing.js [hunk 2]
```diff
@@ -578,7 +602,11 @@ export default class TableResizingModule extends BaseModule {
                 resultWidth = this._minColumnWidth;
             }
 
-            setLineElementsStyleValue($lineElements, 'width', resultWidth);
+            setLineElementsFormat(this, {
+                lineElements: $lineElements,
```
Comment (DokaRus) [inline js/ui/html_editor/modules/tableResizing.js]:
```suggestion
            setLineElementsFormat(this, {
                elements: $lineElements,
```
You already have `line` in the method name

File: js/ui/html_editor/utils/table_helper.js [hunk 3]
```diff
@@ -29,8 +29,13 @@ function hasEmbedContent(module, selection) {
     return !!selection && module.quill.getText(selection).trim().length < selection.length;
 }
 
-function unfixTableWidth($table) {
-    $table.css('width', 'initial');
+function unfixTableWidth($table, tableData) {
+    const unfixValue = 'initial';
+    if(tableData && tableData[0]) {
+        tableData[0].format('tableWidth', unfixValue);
```
Comment (DokaRus) [inline js/ui/html_editor/utils/table_helper.js]:
```suggestion
function unfixTableWidth($table, tableData = []) {
    const unfixValue = 'initial';
    if(tableData.length) {
        tableData[0].format('tableWidth', unfixValue);
```

And it's really necessary to pass all data? 

```suggestion
function unfixTableWidth($table, tableBlot) {
    const unfixValue = 'initial';
    if(tableBlot) {
        tableBlot.format('tableWidth', unfixValue);
```

File: js/ui/html_editor/utils/table_helper.js [hunk 4]
```diff
@@ -52,9 +57,10 @@ function getAutoSizedElements($table, direction = 'horizontal') {
     return result;
 }
 
-function setLineElementsStyleValue($lineElements, property, value) {
-    each($lineElements, (i, element) => {
-        $(element).css(property, value + 'px');
+function setLineElementsFormat(module, { lineElements, property, value }) {
+    each(lineElements, (i, element) => {
+        const cellBlot = module.quill.scroll.find(element);
+        cellBlot.format('cell' + property[0].toUpperCase() + property.substring(1), value + 'px');
```
Comment (DokaRus) [inline js/ui/html_editor/utils/table_helper.js]: You can use `camelize` utility

https://github.com/DevExpress/DevExtreme/blob/ab7c533db49acfd0071de4b9d7072aeeca9e8562/js/core/utils/inflector.js#L28

File: js/ui/html_editor/utils/toolbar_helper.js [hunk 5]
```diff
@@ -109,8 +109,11 @@ function prepareShowFormProperties(module, type) {
         if(!$element?.length) {
             $element = $(getTargetTableNode(module, type));
         }
+        const tableData = module.quill.getModule('table').getTable();
```
Comment (DokaRus) [inline js/ui/html_editor/utils/toolbar_helper.js]:
```suggestion
        const [table, row, cell] = module.quill.getModule('table').getTable() ?? [];
```
And pass further only what you really need

File: js/ui/html_editor/utils/table_helper.js [hunk 6]
```diff
@@ -52,9 +58,10 @@ function getAutoSizedElements($table, direction = 'horizontal') {
     return result;
 }
 
-function setLineElementsStyleValue($lineElements, property, value) {
-    each($lineElements, (i, element) => {
-        $(element).css(property, value + 'px');
+function setLineElementsFormat(module, { elements, property, value }) {
+    each(elements, (i, element) => {
+        const cellBlot = module.quill.scroll.find(element);
+        cellBlot.format('cell' + camelize(property), value + 'px');
```
Comment (DokaRus) [inline js/ui/html_editor/utils/table_helper.js]:
```suggestion
        cellBlot?.format('cell' + camelize(property), value + 'px');
```

**Review by DokaRus — DISMISSED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Pull request merged**